openmetadata-service/src/main/java/org/openmetadata/service/security/AuthenticationCodeFlowHandler.java

```diff
@@ -349,7 +349,7 @@ public void handleLogout(
       if (session != null) {
         LOG.debug("Invalidating the session for logout");
         session.invalidate();
-        httpServletResponse.sendRedirect(serverUrl);
+        httpServletResponse.sendRedirect(serverUrl + "/logout");
```

Comment:
Better to take it as part of configuration, otherwise if this changes that would need code change?

Reply:
It's handled on Frontend so not sure how frequent it may change
Here:
https://github.com/open-metadata/OpenMetadata/blob/55b5001bb5a0729dd3fa7daa26e22ee07120d0d5/openmetadata-ui/src/main/resources/ui/src/constants/constants.ts#L134